fala/templates/macros/forms.html

```diff
@@ -54,9 +54,9 @@
       <div class="form-row {{ kwargs.row_class or '' }}">
         {% if field %}
           {% if field.errors and controlled_by and not controlled_by.errors or field.errors and not controlled_by %}
-            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-input--error govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}">
+            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-input--error govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}" aria-describedby="field-description-postcode">
           {% else %}
-            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}">
+            <input type="text" name="{{ field.name }}" value="{{ field.data or '' }}" class="govuk-input govuk-!-width-one-third {{ field_attrs.class or '' }}" maxlength="{{ kwargs.max_length or '30' }}" id="id_{{ field.name }}" {% if field.name == 'postcode' %}aria-describedby="field-description-postcode"{% endif %}>
```

Comment:
question - do we need to add a comment to say that we should remove this 'conditional if statement' once FALA refresh has been launched?

Reply:
I can add a comment in regards to why this is appended to the this input sure.

In regards to Fala's redesign, this will still be needed unless we gut and refactor everything in terms of generating forms in FALA which is a big piece of work that isn't in the pipelines from what we've planned in terms of these forms.

Reply:
Fair enough, I hadn't realised we weren't going to 'gut out' the old design once we released the new design.

Reply:
This is still part of the new design. It's just an input field.